app/client/src/ce/sagas/moduleInterfaceSagas.ts

```
@@ -0,0 +1,21 @@
+import type { CanvasWidgetsReduxState } from "ee/reducers/entityReducers/canvasWidgetsReducer";
+
+export interface HandleModuleWidgetCreationSagaPayload {
+  addChildPayload: WidgetAddChild;
+  widgets: CanvasWidgetsReduxState;
+}
+
+export function* handleModuleWidgetCreationSaga(
+  props: HandleModuleWidgetCreationSagaPayload,
+) {
+  return props.widgets;
+}
```

Biome (1.9.4)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)